actors/builtin/multisig/multisig_actor.go

```diff
@@ -69,12 +72,7 @@ type ProposeParams struct {
 	Params abi.MethodParams
 }
 
-type ProposeReturn struct {
-	TxnID TxnID
-}
-
-func (a *MultiSigActor) Propose(rt vmr.Runtime, params *ProposeParams) *ProposeReturn {
-	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
+func (a *MultiSigActor) Propose(rt vmr.Runtime, params *ProposeParams) *cbg.CborInt {
```

Comment:
Can we discuss this change?
1.  I added return structures symmetry with params objects, and consistent with service-method code generators everywhere (protobuf etc).
2. There's a CBOR-able big.Int type here already, we don't need to reference cbg directly here.

Reply:
1) Struct wrapping all return values feels wasteful to me, functions should be able to have simple return types, i'd rather not have to define a new type and run codegen for every new function that wants to return an int
2) This isnt a big int

Reply:
Ok, I see the cost of the struct-wrapping bytes on chain. So methods can return serializable value, done.

And then I'm guessing CborInt is just a wrapper to implement the marshal methods around an int. cbg probably has a bunch of utilities like this that are unrelated to code generation but useful. Ok to land this now, but I might attempt to minimise external deps of this code later.